AirCasting/CoreData/CoreDataHook.swift

```diff
@@ -49,11 +62,20 @@ final class CoreDataHook: NSObject, ObservableObject {
         fetchedResultsController.delegate = self
         return fetchedResultsController
     }()
-
-    func setup(selectedSection: DashboardSection) throws {
+    
+    private func refreshSessions() {
+        let sessionEntities = fetchedResultsController.fetchedObjects ?? []
+        let externalSessionEntities = externalSessionsFetchedResultsController.fetchedObjects ?? []
+        sessions = sessionEntities + externalSessionEntities
+        guard case .following = selectedSection else { return }
```

Comment:
is it for setup only? I'm not following (suchar XD). I'm not sure why you would want a section to be `.following` in the method called `refreshSessions()` otherwise.  If you're using it just for setup I think the name might be misleading, if not we can reuse it for different sections, right? So maybe this one should be moved to `func refreshSessions()`?

Reply:
I liked the suchar 😂

Reply:
I am not quite sure I inderstand, but two things:
1. I would like not to touch is as it is outside the scope of this PR
2. We check if the current section if following, if so we would like to returned sessions ordered